coordinator/manifest/manifest_test.go

```diff
@@ -347,3 +348,503 @@ func TestGenerateUsers(t *testing.T) {
+}
+
+func TestTLSTagEqual(t *testing.T) {
+	testCases := map[string]struct {
+		a, b      TLStag
+		wantEqual bool
+	}{
+		"equal minimal": {
+			a:         TLStag{Outgoing: []TLSTagEntry{{Addr: "foo"}}},
+			b:         TLStag{Outgoing: []TLSTagEntry{{Addr: "foo"}}},
+			wantEqual: true,
+		},
+		"equal multiple entries": {
+			a: TLStag{
+				Outgoing: []TLSTagEntry{
+					{Addr: "foo", Port: "123"},
+					{Addr: "bar", Port: "456"},
+					{Addr: "foo", Port: "456"},
+				},
+			},
+			b: TLStag{
+				Outgoing: []TLSTagEntry{
+					{Addr: "bar", Port: "456"},
+					{Addr: "foo", Port: "456"},
+					{Addr: "foo", Port: "123"},
+				},
+			},
+			wantEqual: true,
+		},
+		"equal full": {
+			a: TLStag{
+				Outgoing: []TLSTagEntry{
+					{
+						Addr:              "foo",
+						Port:              "123",
+						Cert:              "cert-1",
+						DisableClientAuth: true,
+					},
+				},
+				Incoming: []TLSTagEntry{
+					{
+						Addr:              "bar",
+						Port:              "456",
+						Cert:              "cert-2",
+						DisableClientAuth: true,
+					},
+				},
+			},
+			b: TLStag{
+				Outgoing: []TLSTagEntry{
+					{
+						Addr:              "foo",
+						Port:              "123",
+						Cert:              "cert-1",
+						DisableClientAuth: true,
+					},
+				},
+				Incoming: []TLSTagEntry{
+					{
+						Addr:              "bar",
+						Port:              "456",
+						Cert:              "cert-2",
+						DisableClientAuth: true,
+					},
+				},
+			},
+			wantEqual: true,
+		},
+		"different outgoing addr": {
+			a: TLStag{Outgoing: []TLSTagEntry{{Addr: "foo"}}},
+			b: TLStag{Outgoing: []TLSTagEntry{{Addr: "bar"}}},
+		},
+		"different incoming addr": {
+			a: TLStag{Incoming: []TLSTagEntry{{Addr: "foo"}}},
+			b: TLStag{Incoming: []TLSTagEntry{{Addr: "bar"}}},
+		},
+		"different outgoing port": {
+			a: TLStag{Outgoing: []TLSTagEntry{{
+				Addr: "foo",
+				Port: "123",
+			}}},
+			b: TLStag{Outgoing: []TLSTagEntry{{
+				Addr: "foo",
+				Port: "456",
+			}}},
+		},
+		"different outgoing cert": {
+			a: TLStag{Outgoing: []TLSTagEntry{{
+				Addr: "foo",
+				Cert: "cert-1",
+			}}},
+			b: TLStag{Outgoing: []TLSTagEntry{{
+				Addr: "foo",
+				Cert: "cert-2",
+			}}},
+		},
+		"different outgoing disable client auth": {
+			a: TLStag{Outgoing: []TLSTagEntry{{
+				Addr:              "foo",
+				DisableClientAuth: true,
+			}}},
+			b: TLStag{Outgoing: []TLSTagEntry{{
+				Addr:              "foo",
+				DisableClientAuth: false,
+			}}},
+		},
+		"different number of outgoing entries": {
+			a: TLStag{Outgoing: []TLSTagEntry{{Addr: "foo"}}},
+			b: TLStag{Outgoing: []TLSTagEntry{{Addr: "foo"}, {Addr: "bar"}}},
+		},
+		"different number of incoming entries": {
+			a: TLStag{Incoming: []TLSTagEntry{{Addr: "foo"}}},
+			b: TLStag{Incoming: []TLSTagEntry{{Addr: "foo"}, {Addr: "bar"}}},
+		},
+	}
+
+	for name, tc := range testCases {
+		t.Run(name, func(t *testing.T) {
+			assert.Equal(t, tc.wantEqual, tc.a.Equal(tc.b))
```

Comment:
```suggestion
			assert.Equal(t, tc.wantEqual, tc.a.Equal(tc.b))
			assert.Equal(t, tc.wantEqual, tc.b.Equal(tc.a))
```
You may do the same in these tests, too.